sys/clang-format-diff.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python2
```

Comment:
Don't know if you are interested in porting to Python 3, but I am pretty sure you would just need to change this "shebang" line, `import functools`, and change `reduce` to `functools.reduce`

Reply:
already done

Reply:
Thanks :)